### PR TITLE
Quiz app

### PR DESCRIPTION
**Pull request by senali-d**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Build a React application for a quiz play. The quiz should have multiple questions and multiple answers.

Fixes # (#710)

## Type of change

- [x] New Play

# How Has This Been Tested?

This has been tested locally.

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by koustov — CHANGES_REQUESTED** [review]
Few ovservations
1. You might want to highlight the question when its hovered and selected
2. Same for options
3. Start Again doesn't work

**Comment by senali-d** [comment]
> Few ovservations
> 
> 1. You might want to highlight the question when its hovered and selected
> 2. Same for options
> 3. Start Again doesn't work

@koustov Resolved the requested changes.

File: src/plays/quiz-app/QuizApp.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useState } from 'react'
+import PlayHeader from 'common/playlists/PlayHeader';
+
+import { CgEditFlipH } from 'react-icons/cg';
+import { AiOutlineClose, AiOutlineCheck } from 'react-icons/ai'
+import useLocalStorage from 'common/hooks/useLocalStorage';
+
+const questionsArray = [
+  {qNo: 1, title: 'JavaScript ', question: 'Which type of JavaScript language is ___', answers: ['Object-Oriented', 'Object-Based', 'Assembly-language', 'High-level'], answer: 2, qStatus: 0},
```
Comment (Angryman18) [inline src/plays/quiz-app/QuizApp.tsx:10]: good to have a seperate file for this data

**Review by Angryman18 — APPROVED** [review]
looks good.

**Comment by atapas** [comment]
@Angryman18 Can you please resolve the conversation so that I can merge?

**Comment by Angryman18** [comment]
resolved

**Pull request merged**